concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -2171,9 +2169,9 @@ jobs:
                   cd paas-cf/tools/paas_dashboard
                   cf push --no-start paas-dashboard
                   echo Setting DD_API_KEY...
-                  cf set-env paas-dashboard DD_API_KEY {{datadog_api_key}} > /dev/null
+                  cf set-env paas-dashboard DD_API_KEY ((datadog_api_key)) > /dev/null
```

Comment:
we should add quotes here (do we need single quotes in this case?)

Reply:
Good spot.